extra/docs_landing.html

```diff
@@ -1,238 +1,366 @@
-      <path d="M317.707 345.707C318.098 345.317 318.098 344.683 317.707 344.293L311.343 337.929C310.953 337.538 310.319 337.538 309.929 337.929C309.538 338.319 309.538 338.953 309.929 339.343L315.586 345L309.929 350.657C309.538 351.047 309.538 351.681 309.929 352.071C310.319 352.462 310.953 352.462 311.343 352.071L317.707 345.707ZM229 346H317V344H229V346Z" fill="#BD271E"/>
-      <path d="M316.707 255.707C317.098 255.317 317.098 254.683 316.707 254.293L310.343 247.929C309.953 247.538 309.319 247.538 308.929 247.929C308.538 248.319 308.538 248.953 308.929 249.343L314.586 255L308.929 260.657C308.538 261.047 308.538 261.681 308.929 262.071C309.319 262.462 309.953 262.462 310.343 262.071L316.707 255.707ZM228 256L316 256V254L228 254V256Z" fill="#BD271E"/>
-      <path d="M789.293 254.293C788.902 254.683 788.902 255.317 789.293 255.707L795.657 262.071C796.047 262.462 796.681 262.462 797.071 262.071C797.462 261.681 797.462 261.047 797.071 260.657L791.414 255L797.071 249.343C797.462 248.953 797.462 248.319 797.071 247.929C796.681 247.538 796.047 247.538 795.657 247.929L789.293 254.293ZM790 256L904 256V254L790 254V256Z" fill="#BD271E"/>
-      <path d="M789.293 382.293C788.902 382.683 788.902 383.317 789.293 383.707L795.657 390.071C796.047 390.462 796.681 390.462 797.071 390.071C797.462 389.681 797.462 389.047 797.071 388.657L791.414 383L797.071 377.343C797.462 376.953 797.462 376.319 797.071 375.929C796.681 375.538 796.047 375.538 795.657 375.929L789.293 382.293ZM790 384L847 384L847 382L790 382L790 384ZM847 384L904 384L904 382L847 382L847 384Z" fill="#BD271E"/>
-      <path d="M789.293 485.293C788.902 485.683 788.902 486.317 789.293 486.707L795.657 493.071C796.047 493.462 796.681 493.462 797.071 493.071C797.462 492.681 797.462 492.047 797.071 491.657L791.414 486L797.071 480.343C797.462 479.953 797.462 479.319 797.071 478.929C796.681 478.538 796.047 478.538 795.657 478.929L789.293 485.293ZM790 487L847 487L847 485L790 485L790 487ZM847 487L904 487L904 485L847 485L847 487Z" fill="#BD271E"/>
-      <path d="M993.707 419.293C993.317 418.902 992.683 418.902 992.293 419.293L985.929 425.657C985.538 426.047 985.538 426.681 985.929 427.071C986.319 427.462 986.953 427.462 987.343 427.071L993 421.414L998.657 427.071C999.047 427.462 999.681 427.462 1000.07 427.071C1000.46 426.681 1000.46 426.047 1000.07 425.657L993.707 419.293ZM992 420V440H994V420H992Z" fill="#BD271E"/>
-      <text fill="black" xml:space="preserve" style="white-space: pre" font-family="Inter" font-size="26" font-weight="bold" letter-spacing="0em"><tspan x="9" y="24.9545">Components of the Elastic Stack</tspan></text>
-      </svg>
-      
+<div class="row my-4">
```

Comment:
I think overall this looks great, but I'd love to see some of this whitespace removed from this class.
<img width="1373" alt="Screen Shot 2022-11-16 at 11 55 18 AM" src="https://user-images.githubusercontent.com/5618806/202281191-18625096-f40f-498b-86bf-2b37a2602906.png">